api/datalake_api/querier.py

```diff
@@ -222,6 +223,7 @@ def _add_cursor_conditions(self, kwargs, cursor):
             kwargs["ExpressionAttributeNames"]["#n2"] = "metadata"
             kwargs["ExpressionAttributeNames"]["#n3"] = "id"
             kwargs["ExpressionAttributeValues"][":v2"] = cursor.last_id
+            pass
```

Comment:
Hmm, why is this necessary?

Reply:
Ha. No. It's an artifact from some debugging. Good catch.